airflow/providers/qdrant/provider.yaml

```diff
@@ -40,7 +40,7 @@ integrations:
     tags: [software]
 
 dependencies:
-  - qdrant_client>=1.9.0
+  - qdrant_client>=1.10.1
```

Comment:
Same question as
https://github.com/apache/airflow/pull/40707#discussion_r1673116729

What is the motivation to prevent users using 1.9 should they like to? 1.10.x is supported and the default of the Airflow published constraints.

Reply:
One reported issue we run into is, users use the latest Qdrant image(qdrant/qdrant:v1.10.1) but with older client versions. Though they are intended to be compatible, we sometimes get serialization/deserialization errors when the gap is bigger. Like server on v1.10 and client on v1.7.

Reply:
If you guys agree we can do these bumps less frequently. What do you think?

Like for every 2 or minor versions. Instead of every other.

Reply:
Rebased it because there were some errors and I want to see if rebasing fixes them